<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Unofficial Project Altis Launcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Unofficial Toontown Project Altis launcher&lt;/p&gt;&lt;p&gt;Use at your own risk&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>@Mr_Hugo#6146</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="usernameEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="passwordEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="loginButton">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
